CMakeLists.txt

```diff
@@ -118,6 +118,7 @@ add_subdirectory(protobuf)
 add_subdirectory(schemachange)
 add_subdirectory(sockpool)
 add_subdirectory(sqlite)
+add_subdirectory(tcl)
```

Comment:
We should either document tcldev as a dependency, or make this optional.

Reply:
Good point, I'll need to figure out how to accomplish that with CMake.